samples/nrf9160/aws_fota/README.rst

```diff
@@ -31,56 +112,133 @@ Building and running
-
-Below is an example of a very permissive policy which should be used for **testing only**.
-You can attach it to the certificates if you are unsure what policies are needed for your device.
-To create a policy, go to `AWS IoT Console <https://console.aws.amazon.com/iot/home>`_ and select **Secure->Policies->Create**.
-
-.. code-block:: javascript
-
-    {
-      "Version": "2012-10-17",
-      "Statement": [
-          {
-            "Effect": "Allow",
-            "Action": "iot:*",
-            "Resource": "*"
-          }
-       ]
-     }
-
-To create a thing, log in to the `AWS IoT Console <https://console.aws.amazon.com/iot/home>`_ and select **Manage->Things->Create->Create a single thing**.
-Provide it with a name which corresponds to your device's MQTT Client ID.
-Next, choose to either create certificates or add your own certificates and make sure to **Activate** them.
-Also, make sure to download one of the `root CA certificates <https://docs.aws.amazon.com/iot/latest/developerguide/managing-device-certs.html#server-authentication>`_ and the generated certificates if you choose to let AWS generate some for you.
-
-Then, attach the policy you created earlier to the certificates. You must also add the information from the Root CA, Public Certificate, and Private Key into :file:`certificates.h`.
-
-When these steps are done, deselect ``CONFIG_USE_PROVISIONED_CERTIFICATES`` in menuconfig.
-This will write your new certificates to the secure tag :c:type:`sec_tag_t` defined in menuconfig ``CONFIG_CLOUD_CERT_SEC_TAG`` into the modem.
-
-
-Usage
-*****
-
-1. Configure the host name to be the AWS IoT MQTT Broker endpoint you want to use in menuconfig.
-#. Make sure you have provisioned certificates onto your device. See the steps in `prerequisites`_.
-#. Flash the sample onto the board.
-#. When the sample has successfully connected to the AWS IoT broker, log in to your `AWS IoT Console <https://console.aws.amazon.com/iot/home>`_ and go to **Manage->Jobs->Create**.
-#. Then, choose **Create custom job**, provide it with a unique **Job ID**, and select the **Device to update**.
-#. Add a **Job file**.
-
-.. note::
-   * An example of a **job file** can be found in the documentation of the :ref:`lib_aws_fota` library.
-     You can upload it to an `S3 bucket <https://console.aws.amazon.com/s3>`_ on your AWS account and select it from there. If you use the AWS IoT Console, this is the only way to add a job file to a custom job.
-
-
-7. Choose **Snapshot** as job type and press **Next**.
+.. _configuring:
+
+Sample configuration
+====================
+
+Before you build the sample, check and update the following configuration options:
+
+Application version:
+   The version string is printed when the sample starts.
+   (?not implemented yet?)
```

Comment:
This was added in https://github.com/NordicPlayground/fw-nrfconnect-nrf/commit/e06166673182545fcf95daee4492eea7ac20cdf1